admin/class-pointers.php

```diff
@@ -281,9 +228,9 @@ function print_scripts( $selector, $options, $button_array ) {
 
 				function wpseo_store_answer(input, nonce) {
```

Comment:
We should consider to put this function in a javascript file, because it's javascript